experimental/saiexperimentaldash.h

```diff
@@ -0,0 +1,1765 @@
+     * @default SAI_PA_VALIDATION_ENTRY_ACTION_PERMIT
+     */
+    SAI_PA_VALIDATION_ENTRY_ATTR_ACTION = SAI_PA_VALIDATION_ENTRY_ATTR_START,
+
+    /**
+     * @brief End of attributes
+     */
+    SAI_PA_VALIDATION_ENTRY_ATTR_END,
+
+    /** Custom range base value */
+    SAI_PA_VALIDATION_ENTRY_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /** End of custom range base */
+    SAI_PA_VALIDATION_ENTRY_ATTR_CUSTOM_RANGE_END,
+
+} sai_pa_validation_entry_attr_t;
+
+/**
+ * @brief Entry for outbound_eni_lookup_from_vm_entry
+ */
+typedef struct _sai_outbound_eni_lookup_from_vm_entry_t
+{
+    /**
+     * @brief Switch ID
+     *
+     * @objects SAI_OBJECT_TYPE_SWITCH
+     */
+    sai_object_id_t switch_id;
+
+    /**
+     * @brief Exact matched key src_mac
+     */
+    sai_mac_t src_mac;
+
+} sai_outbound_eni_lookup_from_vm_entry_t;
+
+/**
+ * @brief Attribute ID for outbound_eni_lookup_from_vm_entry
+ */
+typedef enum _sai_outbound_eni_lookup_from_vm_entry_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_START,
+
+    /**
+     * @brief Action set_eni parameter ENI
+     *
+     * @type sai_uint16_t
+     * @flags CREATE_AND_SET
+     * @isvlan false
+     * @default 0
+     */
+    SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_ENI = SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_START,
+
+    /**
+     * @brief End of attributes
+     */
+    SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_END,
+
+    /** Custom range base value */
+    SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /** End of custom range base */
+    SAI_OUTBOUND_ENI_LOOKUP_FROM_VM_ENTRY_ATTR_CUSTOM_RANGE_END,
+
+} sai_outbound_eni_lookup_from_vm_entry_attr_t;
+
+/**
+ * @brief Entry for outbound_eni_to_vni_entry
+ */
+typedef struct _sai_outbound_eni_to_vni_entry_t
+{
+    /**
+     * @brief Switch ID
+     *
+     * @objects SAI_OBJECT_TYPE_SWITCH
+     */
+    sai_object_id_t switch_id;
+
+    /**
+     * @brief Exact matched key ENI
+     */
+    sai_uint8_t ENI;
+
+} sai_outbound_eni_to_vni_entry_t;
+
+/**
+ * @brief Attribute ID for outbound_eni_to_vni_entry
+ */
+typedef enum _sai_outbound_eni_to_vni_entry_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_START,
+
+    /**
+     * @brief Action set_vni parameter VNI
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_VNI = SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_START,
+
+    /**
+     * @brief End of attributes
+     */
+    SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_END,
+
+    /** Custom range base value */
+    SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /** End of custom range base */
+    SAI_OUTBOUND_ENI_TO_VNI_ENTRY_ATTR_CUSTOM_RANGE_END,
+
+} sai_outbound_eni_to_vni_entry_attr_t;
+
+/**
+ * @brief Attribute ID for dash_acl
+ */
+typedef enum _sai_dash_acl_attr_t
```

Comment:
I think we may want to have ACL rule and group separated and this would be `sai_dash_acl_rule_attr_t`